QuasarHelios2015/src/org/team1540/quasarhelios/Pressure.java

```diff
@@ -14,12 +17,23 @@
 	public static final BooleanOutput compressor = Igneous.usePCMCompressor();
 	
 	public static void setup() {
+		FloatInputPoll pressureInput = Igneous.makeAnalogInput(0);
 		
-		pressureGauge = FloatMixing.createDispatch(FloatMixing.normalizeFloat(Igneous.makeAnalogInput(0), 
-				ControlInterface.mainTuning.getFloat("main-pressure-min", 0.0f), ControlInterface.mainTuning.getFloat("pressure-max", 1.0f)), Igneous.globalPeriodic);
+		FloatStatus min = ControlInterface.mainTuning.getFloat("pressure-min", 0.0f);
+		FloatStatus max = ControlInterface.mainTuning.getFloat("pressure-max", 1.0f);
+		
+		EventStatus setMin = new EventStatus();
+		EventStatus setMax = new EventStatus();
+		
+		FloatMixing.pumpWhen(setMin, pressureInput, min);
+		FloatMixing.pumpWhen(setMax, pressureInput, max);
+		
+		pressureGauge = FloatMixing.createDispatch(FloatMixing.normalizeFloat(pressureInput, min, max), Igneous.globalPeriodic);
 		
 		Cluck.publish("Pressure Switch", pressureSwitch);
 		Cluck.publish("Pressure Gauge", pressureGauge);
 		Cluck.publish("Compressor Enable", compressor);
+		Cluck.publish("Pressure Min Set", setMin);
+		Cluck.publish("Pressure Max Set", setMax);
```

Comment:
Any particular reason that you published an `EventStatus` instead of publishing an `EventOutput` directly? Remember that `pumpEvent` exists.